ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -694,6 +694,65 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- pump_speed_frequency_setpoint
+- condensing_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- condensing_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- oil_heater_status:
+  - ON
+  - OFF
+- inhibit_start_time_status
+- limit_percentage_capacity
+- total_compressor_count
+- compressor_run_time_sensor
+- guidevane_position_percentage_sensor
+- oil_temperature_sensor
+- oil_pump_differential_pressure_sensor
+- exhaust_gas_compressor_temperature_sensor
+- bearing_temperature_sensor
+- winding_temperature_sensor
+- start_mode:
+  - OFF
+  - ON
+- compressor_temperature_setpoint
+- compressor_percentage_setpoint
```

Comment:
`speed_percentage_command` i believe is the appropriate field here.

Reply:
i have created a new subfield called load, and change the name to load_compressor_percentage_setpoint

Reply:
field has been removed - resolving